packages/app/src/components/DBTimeChart.tsx

```diff
@@ -245,12 +248,21 @@ function DBTimeChartComponent({
     [config, granularity],
   );
 
+  const { data: me } = api.useMe();
   const { data, isLoading, isError, error, isPlaceholderData, isSuccess } =
     useQueriedChartConfig(queriedConfig, {
       placeholderData: (prev: any) => prev,
-      queryKey: [queryKeyPrefix, queriedConfig, 'chunked'],
+      queryKey: [
+        queryKeyPrefix,
+        queriedConfig,
+        'chunked',
+        disableQueryChunking,
+        enableParallelQueries && me?.team?.parallelizeWhenPossible,
+      ],
       enabled,
       enableQueryChunking: !disableQueryChunking,
+      enableParallelQueries:
+        enableParallelQueries && me?.team?.parallelizeWhenPossible,
     });
```

Comment:
This is my bad, I should have left a comment in the code here, but we should ensure that the query key and options here match the ones in `useSearchTotalCount` - otherwise we'll issue separate queries to get the total count of search results, even though we should be able to use the same query results for both the total count and the histogram.